webpack.config.js

```diff
@@ -0,0 +1,24 @@
+var webpack = require('webpack')
+
+
+module.exports = {
+  entry: './main.js',
+
+  output: {
+    path: "public",
+    filename: 'bundle.js',
+    publicPath: '/'
+  },
+
+  module: {
+    loaders: [
+      { test: /\.js$/, exclude: /node_modules/, loader: 'babel-loader?presets[]=es2015&presets[]=react' }
+    ]
+  },
+
```

Comment:
wait.. you mean you didn't use react-hot? Awh man.

https://github.com/gaearon/react-hot-loader